src/Clause.ts

```diff
@@ -51,16 +52,85 @@ export default class Clause extends Builder {
       // <emu-clause id=foo aoid> === <emu-clause id=foo aoid=foo>
       this.aoid = node.id;
     }
+
+    let header = this.node.firstElementChild;
+    while (header != null && header.tagName === 'SPAN' && header.children.length === 0) {
+      // skip oldids
+      header = header.nextElementSibling;
+    }
+    if (header == null || header.tagName !== 'H1') {
```

Comment:
Missing header and header being the wrong tag are kinda different, and I'd prefer to receive a different message for those cases.

Reply:
Done.